src/ARCtrl/ARCtrl.Javascript.fsproj

```diff
@@ -0,0 +1,90 @@
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="Json\Decode.fs" />
+    <Compile Include="Json\Encode.fs" />
+    <Compile Include="Json\Comment.fs" />
+    <Compile Include="Json\OntologyAnnotation.fs" />
+    <Compile Include="Json\OntologySourceReference.fs" />
+    <Compile Include="Json\DataFile.fs" />
+    <Compile Include="Json\Data.fs" />
+    <Compile Include="Json\Person.fs" />
+    <Compile Include="Json\Publication.fs" />
+    <Compile Include="Json\Process\Value.fs" />
+    <Compile Include="Json\Process\Factor.fs" />
+    <Compile Include="Json\Process\FactorValue.fs" />
+    <Compile Include="Json\Process\ProtocolParameter.fs" />
+    <Compile Include="Json\Process\MaterialType.fs" />
+    <Compile Include="Json\Process\MaterialAttribute.fs" />
+    <Compile Include="Json\Process\Component.fs" />
+    <Compile Include="Json\Process\Protocol.fs" />
+    <Compile Include="Json\Process\MaterialAttributeValue.fs" />
+    <Compile Include="Json\Process\Material.fs" />
+    <Compile Include="Json\Process\Source.fs" />
+    <Compile Include="Json\Process\Sample.fs" />
+    <Compile Include="Json\Process\ProcessParameterValue.fs" />
+    <Compile Include="Json\Process\ProcessInput.fs" />
+    <Compile Include="Json\Process\ProcessOutput.fs" />
+    <Compile Include="Json\Process\Process.fs" />
+    <Compile Include="Json\Process\ProcessSequence.fs" />
+    <Compile Include="Json\Process\StudyMaterials.fs" />
+    <Compile Include="Json\Table\Compression.fs" />
+    <Compile Include="Json\Table\CompositeCell.fs" />
+    <Compile Include="Json\Table\IOType.fs" />
+    <Compile Include="Json\Table\CompositeHeader.fs" />
+    <Compile Include="Json\Table\ArcTable.fs" />
+    <Compile Include="Json\Table\Templates.fs" />
+    <Compile Include="Json\DataMap\DataMap.fs" />
+    <Compile Include="Json\Assay.fs" />
+    <Compile Include="Json\Study.fs" />
+    <Compile Include="Json\Investigation.fs" />
+    <Compile Include="Json\ROCrateObject.fs" />
+    <Compile Include="Json\ARC.fs" />
+    <Compile Include="WebRequest\WebRequest.Node.fs" />
+    <Compile Include="WebRequest\WebRequest.fs" />
+    <Compile Include="Template.Web.fs" />
+    <Compile Include="ARC.fs" />
+    <Compile Include="Json.fs" />
+    <Compile Include="Xlsx.fs" />
+    <None Include="README.md" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Fable.Fetch"/>
+    <PackageReference Include="Fable.SimpleHttp"/>
+    <PackageReference Include="Thoth.Json.Javascript"/>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\Contract\ARCtrl.Contract.fsproj" />
+    <ProjectReference Include="..\CWL\ARCtrl.CWL.fsproj" />
+    <ProjectReference Include="..\FileSystem\ARCtrl.FileSystem.fsproj" />
+    <ProjectReference Include="..\Json\ARCtrl.Json.fsproj" />
+    <ProjectReference Include="..\Spreadsheet\ARCtrl.Spreadsheet.fsproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="*.fsproj; **\*.fs; **\*.fsi" PackagePath="fable\" />
```

Comment:
Using [Fable.Package.SDK](https://github.com/fable-compiler/) allows to remove these lines and will asked you to set the `PackageTags` information so your NuGet package can be listed in https://fable.io/packages/

Reply:
This did not work for me out of the box, will open an Issue in the given project.

Reply:
https://github.com/fslaborg/FsSpreadsheet/blob/e59da935ebced63c3bcd7ed9e9dd3e276715fe54/src/FsSpreadsheet/FsSpreadsheet.fsproj#L6

Here are the lines in question. I thought this would be enough to trigger the inclusion of the src files in a Fable folders when running `dotnet pack`

Reply:
It should be enough yes,

Use the following to include the dependency, you didn't have it included by default because it is only done automatically since 1.1.0. This will make your package not include Fable.Package.SDK in its dependencies.

```xml
<PackageReference Include="Fable.Package.SDK">
  <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
  <PrivateAssets>all</PrivateAssets>
</PackageReference>
```

You should also remove:

```xml
<Content Include="*.fsproj; **\*.fs; **\*.fsi" PackagePath="fable\" />
```

If this is still not working I can have a look at it if you want

Reply:
Hey, sry forgot to write here. The Inclusion of the Source Files works now!

Just got another small issue: https://github.com/fable-compiler/Fable.Package.SDK/issues/6

It does produce a proper nuget file. So ignoring the error kinda works, but would be nice to find the reason behind this.

Reply:
Its strange in theory if there is an error it should fails.

But perhaps MSBuild create the package before and then the CLI exist with an error code. I will check the issue you reported.

Reply:
The issue is caused by this line:

```xml
<PackageTags>ARC F# FSharp dotnet .Net bioinformatics biology fable-library datascience dataplant nfdi metadata</PackageTags>
```

Judging by the content of the tag, I think you wanted to use `<Description>`

Reply:
> Its strange in theory if there is an error it should fails.
> 
> But perhaps MSBuild create the package before and then the CLI exist with an error code. I will check the issue you reported.

Yeah sry with "ignoring the error" I meant actually catching *and ignoring* the error :D

Reply:
> The issue is caused by this line:
> 
> ```
> <PackageTags>ARC F# FSharp dotnet .Net bioinformatics biology fable-library datascience dataplant nfdi metadata</PackageTags>
> ```
> 
> Judging by the content of the tag, I think you wanted to use `<Description>`

![image](https://github.com/user-attachments/assets/24ad2f79-0a43-4a60-8b8a-daca6d620403)

These are actually tags used by nuget to increase findability of the package.

Reply:
Kinda hotfixed it by merging the two lists (See issue)